stripe/src/main/java/com/stripe/android/paymentsheet/BasePaymentMethodsListFragment.kt

```diff
@@ -61,6 +69,11 @@ internal abstract class BasePaymentMethodsListFragment(
         eventReporter.onShowExistingPaymentOptions()
 
         viewBinding.header.text = createHeaderText()
+
```

Comment:
It might be worth a quick discussion with @mshafrir-stripe on the difference between processing and the animated button.  Seems like they have some overlapping functionality/purpose.

Here are a couple of things that might be worth thinking  about:
* What does updateRootViewClickHandling do?  Seems like it is trying to disable and click handling?  Is that working as intended?
* Can we skip the processing variable and just use the viewState?
* Should checkout set the processing state earlier? - before the google pay stuff?
* What is ctaEnabled looks like it builds off the processing state among other things.

Reply:
+1 to disabling the carousel and its item views

Reply:
Fixed the processing state for the custom add card flow, wrote down follow up task for refactoring